lib/nerves_hub_link/configurator.ex

```diff
@@ -93,4 +109,9 @@ defmodule NervesHubLink.Configurator do
 
     %{config | fwup_public_keys: config.fwup_public_keys ++ fwup_public_keys}
   end
+
+  @spec maybe_add_fwup_devpath(Config.t()) :: Config.t()
+  defp maybe_add_fwup_devpath(config) do
```

Comment:
This doesn't feel like a "maybe" unless I'm missing something.

Reply:
Whoops, that was supposed to be `put_new` . just a sec